server/manager/src/main/java/org/apache/accumulo/manager/tableOps/split/PreSplit.java

```diff
@@ -76,79 +64,21 @@ public long isReady(long tid, Manager manager) throws Exception {
+    if (canStart(tid, manager)) {
       return 0;
-    } else if (opid.equals(tabletMetadata.getOperationId())) {
-      if (tabletMetadata.getLocation() == null) {
-        // the operation id is set and there is no location, so can proceed to split
-        return 0;
-      } else {
-        // the operation id was set, but a location is also set wait for it be unset
-        return 1000;
-      }
-    } else {
-      try (var tabletsMutator = manager.getContext().getAmple().conditionallyMutateTablets()) {
-
-        tabletsMutator.mutateTablet(splitInfo.getOriginal()).requireAbsentOperation()
-            .requireSame(tabletMetadata, LOCATION, PREV_ROW).putOperation(opid)
-            .submit(tmeta -> opid.equals(tmeta.getOperationId()));
-
-        Map<KeyExtent,Ample.ConditionalResult> results = tabletsMutator.process();
-        if (results.get(splitInfo.getOriginal()).getStatus() == Status.ACCEPTED) {
-          log.trace("Successfully set operation id for split {}", FateTxId.formatTid(tid));
-          if (tabletMetadata.getLocation() == null) {
-            // the operation id was set and there is no location, so can move on
-            return 0;
-          } else {
-            // now that the operation id set, generate an event to unload the tablet
-            manager.getEventCoordinator().event(splitInfo.getOriginal(),
-                "Set operation id %s on tablet for split", FateTxId.formatTid(tid));
-            // the operation id was set, but a location is also set wait for it be unset
-            return 1000;
-          }
-        } else {
-          log.trace("Failed to set operation id for split {}", FateTxId.formatTid(tid));
-          // something changed with the tablet, so setting the operation id failed. Try again later
-          return 1000;
-        }
-      }
     }
+    return 1000;
   }
 
   @Override
   public Repo<Manager> call(long tid, Manager manager) throws Exception {
 
-    manager.getSplitter().removeSplitStarting(splitInfo.getOriginal());
-
-    TabletMetadata tabletMetadata = manager.getContext().getAmple()
-        .readTablet(splitInfo.getOriginal(), PREV_ROW, LOCATION, OPID);
-
-    var opid = TabletOperationId.from(TabletOperationType.SPLITTING, tid);
-
-    if (tabletMetadata == null || !opid.equals(tabletMetadata.getOperationId())) {
-      // the tablet no longer exists or we could not set the operation id, maybe another operation
-      // was running, lets not proceed with the split.
-      var optMeta = Optional.ofNullable(tabletMetadata);
-      log.trace("{} Not proceeding with split. extent:{} location:{} opid:{}",
-          FateTxId.formatTid(tid), splitInfo.getOriginal(),
-          optMeta.map(TabletMetadata::getLocation).orElse(null),
-          optMeta.map(TabletMetadata::getOperationId).orElse(null));
-      return null;
+    if (!canContinue(tid, manager)) {
+      throw new IllegalStateException(
+          "Tablet is in an unexpected condition: " + splitInfo.getOriginal());
```

Comment:
This is changing the behavior, its throwing exceptions under different conditions.